javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-playlist-app/scripts.js

```diff
@@ -0,0 +1,202 @@
+    { title: "Scratching The Surface", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg",artist: "Quincy Larson", duration: "4:25", src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/Scratching%20The%20Surface.mp3"},
+    { title: "Can't Stay Down", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg", artist: "Quincy Larson", duration: "4:15" , src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/Can't%20Stay%20Down.mp3"},
+    { title: "Still Learning", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg", artist: "Quincy Larson", duration: "3:51" , src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/Still%20Learning.mp3"},
+    { title: "Cruising for a Musing", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg", artist: "Quincy Larson", duration: "3:34" , src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/Cruising%20for%20a%20Musing.mp3"},
+    { title: "Never Not Favored", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg", artist: "Quincy Larson", duration: "3:35" , src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/Never%20Not%20Favored.mp3"},
+    { title: "From the Ground Up", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg", artist: "Quincy Larson", duration: "3:12" , src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/From%20the%20Ground%20Up.mp3"},
+    { title: "Walking on Air", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg", artist: "Quincy Larson", duration: "3:25" , src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/Walking%20on%20Air.mp3"},
+    { title: "Can't Stop Me. Can't Even Slow Me Down.", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg", artist: "Quincy Larson", duration: "3:52" , src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/Can't%20Stop%20Me.%20Can't%20Even%20Slow%20Me%20Down.mp3"},
+    { title: "The Surest Way Out is Through", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg", artist: "Quincy Larson", duration: "3:10" , src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/The%20Surest%20Way%20Out%20is%20Through.mp3"},
+    { title: "Chasing That Feeling", cover: "https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg", artist: "Quincy Larson", duration: "2:43" , src:"https://cdn.freecodecamp.org/MP3%20Player%20Project/Chasing%20That%20Feeling.mp3"},
+  ];
+
+  // song playlist
+const songListDiv = document.getElementById("playlist__songs");
+const songPlaylistDiv = document.querySelector(".playlist");
+
+function renderSongs(songs) {
+
+// map Array Method
+// TODO: First Array for steps
+const songList = songs.map((song) => {
```

Comment:
```suggestion
const songList = songs.map((song, index) => {
```

See the comment on the button below for an explanation.